packages/gatsby-dev-cli/src/watch.js

```diff
@@ -103,19 +113,30 @@ function watch(
     ? _.intersection(packages, seenPackages)
     : seenPackages
 
-  let packagesToInstall = []
+  let ignoredPackageJSON = new Map()
+  const ignorePackageJSONChanges = (packageName, contentArray) => {
+    ignoredPackageJSON.set(packageName, contentArray)
 
-  const getPackagesToInstall = packages =>
-    packages.forEach(pkg => {
-      if (localPackages.includes(pkg)) {
-        packagesToInstall.push(pkg)
-      }
-      if (depTree[pkg]) {
-        getPackagesToInstall([...depTree[pkg]])
-      }
-    })
+    return () => {
+      ignoredPackageJSON.delete(ignoredPackageJSON)
```

Comment:
Shouldn't this be 
```suggestion
      ignoredPackageJSON.delete(packageName)
```

Reply:
Oh yeah! It was like that before and wasn't caught by anyone ops